roles/confluent.variables/defaults/main.yml

```
@@ -1101,9 +1106,372 @@ control_center_health_check_user: "{{control_center_ldap_user}}"
 ### Password for authenticated Control Center Health Check. Set if using customized security like Basic Auth.
 control_center_health_check_password: "{{control_center_ldap_password}}"
 
+
+# Kafka Connect Replicator Variables
+
+## Set this variable to customize the Linux User that the Kafka Connect Replicator Service runs with. Default user is cp-kafka-connect-replicator.
+kafka_connect_replicator_user: "{{kafka_connect_replicator_default_user}}"
+
+### Set this variable to customize the Linux Group that the Kafka Connect Replicator Service user belongs to. Default group is confluent.
+kafka_connect_replicator_group: "{{kafka_connect_replicator_default_group}}"
+
+### Variable to define bootstrap servers for Kafka Connect Replicator.  Mandatory for Kafka Connect Replicator setup.
+kafka_connect_replicator_bootstrap_servers: localhost:9092
+
+### Use to set custom Kafka Connect Replicator properties. This variable is a dictionary. Put values true/false in quotation marks to perserve case.
+kafka_connect_replicator_custom_properties: "{{ kafka_connect_replicator.properties }}"
```

```suggestion
kafka_connect_replicator_custom_properties: {}
```

The <component>.properties dicts were the old way of doing properties, its a deprecated variable that we dont need to create for this component

---

Aren't there supposed to be custom properties for each config file? You should probably add unit tests to validate each of those dictionary merges works